libr/core/core.c

```diff
@@ -2396,7 +2396,7 @@ R_API RCore *r_core_fini(RCore *c) {
 	R_FREE (c->cmdlog);
 	r_th_lock_free (c->lock);
 	R_FREE (c->lastsearch);
-	c->cons->pager = NULL;
+	free (c->cons->pager);
```

Comment:
I think this, and also the other `free` should use `R_FREE` instead, to prevent double frees.